api/configuration/v1beta1/kongupstreampolicy_types.go

```diff
@@ -131,6 +139,22 @@ type KongUpstreamHash struct {
 	URICapture *string `json:"uriCapture,omitempty"`
 }
 
+// KongUpstreamStickySessions defines the sticky session configuration for Kong upstream.
+// Sticky sessions ensure that requests from the same client are routed to the same backend target.
+// This is achieved using cookies and requires Kong Enterprise Gateway.
+// +apireference:kic:include
+type KongUpstreamStickySessions struct {
+	// Cookie is the name of the cookie to use for sticky sessions.
+	// Kong will generate this cookie if it doesn't exist in the request.
+	// +kubebuilder:validation:MinLength=1
+	Cookie string `json:"cookie"`
```

Comment:
This is required, correct?

```suggestion
	//
	// +required
	// +kubebuilder:validation:MinLength=1
	Cookie string `json:"cookie"`
```